### DISCUSSION TITLE
Springboot 3 and JDK17

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor#3727] [General]
We saw some vulnerabilities with springboot 2x and jdk11. it stopped us from going to production. how can I build conductor in jdk17 and springboot 3.x

**Comment by v1r3n** [comment]
Hi @shekarpcs the latest release updates the JDK version to 17.  SB 3.x is on the roadmap but is a little more involved effort.

**Comment by LuisLainez** [comment]
So we have an update of Spring 2? There are also vulnerabilities in Spring2, and the entire Spring ecosystem and dependencies is ditching Spring 2 in favor of 3 - mostly on the jakarta side. 
Is there anything happening there?

**Comment by LuisLainez** [comment]
So, for all maintainers that are interested - I have started a fork with a branch for this upgrade
https://github.com/LuisLainez/conductor/pull/1
Not so sure how to collaborate in projects like this, as tbh it's my first Open Source project collaboration outside of a company.
I have done some advances into it, but lots more to go. If you want to help, you can find me in the #conductor slack channel, or look for my name Luis Lainez in there - we can use the thread https://orkes-conductor.slack.com/archives/C02KJ820XPW/p1698131436785879

**Reply by LuisLainez** [reply]
We have a PR up https://github.com/Netflix/conductor/pull/3824 - some issues with starting the server in local though.
